src/app/contexts/ToggleContext/index.test.jsx

```diff
@@ -20,16 +25,67 @@ const mockToggleValues = {
   },
 };
 
+const mockToggleValuesRemoteFeatureToglles = {
+  local: {
+    remoteFeatureToggles: {
+      enabled: true,
+    },
+  },
+};
+
+const mockTogglesResponse = {
+  toggles: {
+    toggleName: {
+      enabled: true,
+      value: '',
+    },
+  },
+};
+
 jest.mock('#lib/config/toggles', () => mockToggleValues);
+jest.mock('./utils/constructTogglesEndpoint', () =>
+  jest.fn().mockReturnValue('test.com'),
+);
+
+beforeEach(() => {
+  process.env.SIMORGH_APP_ENV = 'local';
+  global.fetch = fetch;
```

Comment:
what does this line do?

Reply:
This ensures that any call to fetch is caught by the fetchMock package, allowing us to easily mock network requests using the fetch api.